### ISSUE TITLE
Replace `LARGE_INTEGER` and `ULARGE_INTEGER` with `Int64` and `UInt64` respectively

### ISSUE THREAD

**Issue by kennykerr** [microsoft/win32metadata#1485]
These structs exist only to support very old compilers that don't natively support 64-bit integers.

**Comment by kennykerr** [comment]
Yep!

**Comment by kennykerr** [comment]
Both C# and Rust already map this to native compiler types. It would just be nice not to have to do this all over the place.

**Comment by marler8997** [comment]
Couldn't this change have an affect on certain ABIs?  For example the ABI change that occurs on COM calls when the return type is a native type vs a struct?  Maybe there's no examples of this though?

**Comment by kennykerr** [comment]
I don't believe so. Keep in mind that they are unions, not structs. And no, I'm not aware any such cases.

**Comment by marler8997** [comment]
> I don't believe so. Keep in mind that they are unions, not structs.

Didn't you say yourself here that you change the ABI depending on if you're looking at a native typedef?

https://github.com/microsoft/win32metadata/issues/636#issuecomment-1232025001

> I limit it to those methods that return a type that is both a struct and doesn't have the NativeTypedef attribute. In my parser, structs automatically exclude enums and interfaces.

Are you saying unions are treated like primitive types when determing ABI rather than treated as structs would be?

**Comment by kennykerr** [comment]
I just mean that since these are unions, they have a different layout and alignment to structs. Feel free to take it for a spin to see how MSVC behaves. That's ultimately the litmus test for the calling convention issue.
